activity_hub.php

```diff
@@ -298,8 +332,14 @@ function summarize_stage($stage, $desired_states, $show_filtered_projects = fals
         // already included in the following round's numbers.
         // (Use '@' to suppress "Undefined property" notice:
         // not every stage has a 'project_complete_state'.)
+        // SR is the odd-man-out, and the count needs to be pulled
+        // separately from the others since $stage_state is the same
+        // as for "in PP", but needs the smoothread_deadline to limit it.
         if ($stage_state == @$stage->project_complete_state) {
             $count = array_get($n_projects_transitioned_to_state_, $stage_state, 0);
+        } elseif ($stage->id == 'SR') {
```

Comment:
is there a way we could reference https://github.com/DistributedProofreaders/dproofreaders/blob/master/pinc/stages.inc#L584 in same light as stage references?

Reply:
I'm not entirely sure what you're asking. Are you asking, "Do you have to refer to it by name?" If so, the answer is "I don't know, but ..."

SR is an odd bird. It was tacked[**hacked] on as a new stage, but wasn't given a separate state as the proofing rounds and the PP/PPV rounds were. To be in SR, a project must be in `PROJ_POST_FIRST_CHECKED_OUT` and must have a `smoothread_deadline` in the `projects` table that has not expired. That's the only way to uniquely identify it as being available for smooth reading.

I'd love for you to poke around and see if you can suggest something less hacky! :grin:

Reply:
Reading though, it actually doesn't appear we have some global define or constant for the id 'SR', so this seems fine.

Reply:
Aaah, okay, that's what you meant. You know, I'm so used to the awareness that SR is _**not**_ a distinct state, that it never occurred to me to look, and it's always a good idea to check for things like that.